Doc/library/functions.rst

```diff
@@ -1703,18 +1703,19 @@ are always available.  They are listed here in alphabetical order.
 
 
    With three arguments, return a new type object.  This is essentially a
-   dynamic form of the :keyword:`class` statement. The *name* string is the
-   class name and becomes the :attr:`~definition.__name__` attribute; the *bases*
-   tuple itemizes the base classes and becomes the :attr:`~class.__bases__`
-   attribute; and the *dict* dictionary is the namespace containing definitions
-   for class body and is copied to a standard dictionary to become the
-   :attr:`~object.__dict__` attribute.  For example, the following two
-   statements create identical :class:`type` objects:
+   dynamic form of the :keyword:`class` statement. The *name* string is
+   the class name and becomes the :attr:`~definition.__name__` attribute.
+   The *bases* tuple contains the base classes and becomes the
+   :attr:`~class.__bases__` attribute; if empty, :class:`object`, the
+   ultimate base of all classes, is added.  The *dict* dictionary contains
+   attribute and method definitions for the class body; it may be copied
+   or wrapped before becoming the :attr:`~class.__dict__` attribute.
```

Comment:
Where does this link go?  `class.__bases__` is at https://docs.python.org/3/library/stdtypes.html#class.__bases__ but where is `class.__dict__`?

Reply:
No link is created!

Ideas:
- keep `object.__dict__` but amend the doc there (in library/stdtypes) to describe `__dict__` for class objects
- change the reference to be to `__dict__ (class attribute)`

Reply:
Since I changed `object` to `class` and that does not work, I changed it back for this issue, so it continues to link to
file:///F:/dev/3x/Doc/build/html/library/stdtypes.html#object.__dict__
Since 'objects' include 'classes', the current text is not incorrect.  I consider changing it a separate issue.
If classes were the only objects with __dict__ attribute (I am not sure), and object.__dict__ should be class.__dict__, a change would require grepping for other references to change.  This would be a different issue also.